library/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/RumFeature.kt

```diff
@@ -142,6 +144,11 @@ class RumFeature internal constructor(
         backgroundEventTracking = configuration.backgroundEventTracking
         trackFrustrations = configuration.trackFrustrations
         rumEventMapper = configuration.rumEventMapper
```

Comment:
I think `rumEventMapper` is actually not used and can be removed, because when we initialize writer, we do it directly from `configuration.rumEventMapper` https://github.com/DataDog/dd-sdk-android/blob/3db8d5a9981b8dacfce9dcc49bdd8c5ced2604e4/library/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/RumFeature.kt#L195